source/module_esolver/esolver_dp.cpp

```diff
@@ -75,16 +97,31 @@ namespace ModuleESolver
         cell[8] = ucell.latvec.e33 * ucell.lat0_angstrom;
 
         int iat = 0;
+
+        // added by Haocheng 2024/6/2 //
         for (int it = 0; it < ucell.ntype; ++it)
-        {
+        {   
+            #pragma omp parallel for schedule(static)
```

Comment:
Hello, I suggest to change loop of `it-ia` to the loop of `iat` , you can follow this code : https://github.com/deepmodeling/abacus-develop/blob/develop/source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/ekinetic_new.cpp#L108

Reply:
Please review my latest version.